src/backend/app/db/postgis_utils.py

```diff
@@ -777,3 +785,160 @@ def parse_featcol(features: Union[Feature, FeatCol, MultiPolygon, Polygon]):
     elif isinstance(features, Feature):
         feat_col = geojson.FeatureCollection([feat_col])
     return feat_col
+
+
+def request_snapshot(geometry):
```

Comment:
This function and the `poll_task_status` function can be replaced with osm_fieldwork.PostgresClient.
There is already functionality in place to get a data extract using provided config 👍 
Example: https://github.com/hotosm/fmtm/blob/319f65e61e67f7cb4819129f1141703e12e31dad/src/backend/app/projects/project_crud.py#L449